jbt-it/neomdb
# 174 client edit project

Implements a page to display the project details as well as the funtionality to edit the details, or change the project status eg. put it in billing or start the second application phase.

Note: the functionalities are not yet provided, an alert with the respective information will pop up and will be replaced with the actual functionality as soon as the backend is done.

How to test:
- Either open /projects/1 or open the project overview and navigate to "Meine Projekte" and click on the first or second project
- Check the project with ID 1, this should display a project in "Durchführung" below you should see the project data and billing data
- Check the project with ID 2, this should display a project in "Bewerbung" with the applicants in a table below and the respective buttons for the team composition or starting the second application phase
- Click on the edit button and change some values and then save. Note: for the second application phase the idea is to set the start 2 and end 2 date with the respective button

Closes #167
